src/reducers/requests/users.js

```diff
@@ -318,6 +318,16 @@ function switchLogin(state: RequestStatusType = initialRequestState(), action: G
     );
 }
 
+function updateServiceTermsStatus(state: RequestStatusType = initialRequestState(), action: GenericAction): RequestStatusType {
+    return handleRequest(
+        UserTypes.UPDATE_SERVICE_TERMS_STATUS_REQUEST,
+        [UserTypes.RECEIVED_ME, UserTypes.UPDATE_ME_SUCCESS, UserTypes.UPDATE_SERVICE_TERMS_STATUS_SUCCESS],
```

Comment:
Why RECEIVED_ME?

Reply:
It was a mistake. Removed.

(Initially, we were planning to send the updated user as the API response. We had changed that behaviour and I should have removed this.)